docs/fleet/locations.md

```diff
@@ -54,18 +54,30 @@ To move a sub-location to the top level of locations, select **Root** from the *
 A location always belongs to the organization it was created in.
 Members of the organization have access to all locations in the organization by default.
 
-You can share a location beyond its organization by [sharing a location with an additional organization](#share-a-location-with-an-additional-organization).
+You can share a location beyond its organization by sharing a location with an additional organization:
 
 #### Share a location with an additional organization
 
 Share your location with another organization you belong to by selecting the organization from the **Add Organization** dropdown menu and clicking **Share**.
 
+To share your location with an organization you are not a member of, enter the organization ID (a string like `1ab2c3d1-1234-123a-abcd-abcdef123456`) and click **Share**.
```

Comment:
Will it show you how to share it with an org you are a member of?

Reply:
See sentence above this--do you think it needs changing?